Tests/L0/Maven/_suite.ts

```diff
@@ -153,6 +176,7 @@ describe('Maven Suite', function () {
     this.timeout(20000);
 
     before((done) => {
+        Q.longStackSupport = true;
```

Comment:
A debug option that has memory and performance impacts but improves stack trace behaviour.